/aruco_node:
  ros__parameters:
    marker_size: 0.15
    aruco_dictionary_id: DICT_5X5_250
    image_topic: /camera/camera/color/image_raw
    camera_info_topic: /camera/camera/color/camera_info
    camera_frame: camera_color_optical_frame
